source/Government.h

```diff
@@ -118,6 +118,10 @@ class Government {
 
 	bool IsProvokedOnScan() const;
 
+	// Get if ships from this government can go to that system or planet.
+	bool AllowJumpingTo(const System &system) const;
+	bool AllowLandingOn(const Planet &planet) const;
```

Comment:
I think `IsRestrictedFrom` would be better names (they'd have the same name but different arguments). Though, this would require inverting the result everywhere it is used currently.

Reply:
No biggie, changed it.